main.js

```diff
@@ -396,13 +450,30 @@ define(function (require, exports, module) {
     })
   }
   
+  function handleRemoteCarets (data) {
+    if (fromWebPath(data.filePath) !== documentRelativePath) return
```

Comment:
Fine for now, but add a TODO so we can support tracking carets in closed files. (Needs to be done for selections to if I remember correctly.)